resources/public/api/conf/1.0/application.yaml

```diff
@@ -37,39 +52,49 @@ paths:
               type: string
         - name: date
           in: query
-          description: Date used to determine validity. If empty date of submission is used
+          description: |
+            The date used to determine validity. If this field is empty, the date of submission is used. The format is DD-MM-YYYY.
           example: 31-02-2024
           required: false
           explode: true
           schema:
             type: string
       responses:
         '200':
-          description: List of EORI Numbers provided with their authorised status as a boolean
+          description: |
+            Success - Authorised. A list of EORI Numbers provided with their authorised status as boolean = 'true'. 
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Authorisations'
         '400':
-          description: Validation failure
+          description: |
+            Validation failure. The server cannot or will not process the request due to something that is perceived to be a client error (e.g., malformed request syntax, invalid request message framing, or deceptive request routing).
         '403':
-          description: Forbidden
+          description: |
+            Forbidden. The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401 Unauthorized, the client's identity is known to the server.
         '405':
-          description: Method not allowed
+          description: |
+            Method not allowed. The request method is known by the server but is not supported by the target resource. 
         '406':
-          description: Not acceptable
+          description: |
+            Not acceptable. This response is sent when the web server, after performing server-driven content negotiation, doesn't find any content that conforms to the criteria given by the user agent.
         '429':
-          description: Request in excess of rate lime
+          description: |
+            Request in excess of rate limit. The user has sent too many requests in a given amount of time.
         '500':
-          description: Internal Server Error
+          description: |
+            Internal Server Error. The server has encountered a situation it does not know how to handle.
+      x-stoplight:
+        id: l4z9id8uvcvj0
 components:
   schemas:
     Authorisations:
       type: object
       properties:
         date:
           type: string
-          example: 2024-02-31
+          example: '2024-02-31'
```

Comment:
Do we need the single quotes? I'm not sure how this renders but they wouldn't need the quotes in the query string

Reply:
removed single quotes